cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -1413,6 +1413,68 @@ TEST_F(TestLargeStringParquetIO, Basics) {
   this->RoundTripSingleColumn(large_array, large_array, arrow_properties);
 }
 
+using TestDeltaBinaryPacked32ParquetIO = TestParquetIO<::arrow::Int32Type>;
+
+TEST_F(TestDeltaBinaryPacked32ParquetIO, DeltaBinaryPacked) {
```

Comment:
The unit test looks good to me, but can we move them just to `encoding_test.cc`?

Reply:
Sure. Since the tests are identical except for the typing, is it worth the effort to make this a typed test suite, but for just two types?